.gitignore

```diff
@@ -1,5 +1,8 @@
 .DS_Store
 
+.idea/
+.vscode/
```

Comment:
```suggestion
```

It appears that IntelliJ makes frequent modifications to the `.idea` directory, whereas the `.vscode` directory sees fewer changes. Furthermore, our `.vscode` directory includes default configurations to accommodate Yarn's PnP (Plug 'n' Play) install strategy. Therefore, I suggest that we only ignore the `.idea` directory.